js/components/agency_component_preview.jsx

```diff
@@ -8,15 +8,30 @@ import PrettyUrl from './pretty_url';
 import { AgencyComponent } from '../models';
 
 
-function AgencyComponentPreview({ agencyComponent }) {
+function AgencyComponentPreview({ onAgencySelect, agencyComponent, isCentralized }) {
   const description = AgencyComponent.agencyMission(agencyComponent);
   const requestUrl = `/request/agency-component/${agencyComponent.id}/`;
+  const onSelect = () => onAgencySelect(agencyComponent.agency);
 
   return (
     <div className="agency-preview usa-grid-full">
       <div className="usa-width-one-whole">
-        <h2>{agencyComponent.agency.name}</h2>
-        <h3>{agencyComponent.title}</h3>
+        {
+          !isCentralized && (
+            <h2>
+              <span // eslint-disable-line jsx-a11y/no-static-element-interactions
+                className="agency-preview_agency-back"
+                onClick={onSelect}
+                onKeyPress={onSelect}
+                role="button"
+                tabIndex="0"
+              >
+                {agencyComponent.agency.name}
+              </span>
+            </h2>
+          )
+        }
+        <h3>{isCentralized ? agencyComponent.agency.name : agencyComponent.title }</h3>
```

Comment:
To be picky, when this is the agency name, it should be an `h2` with a class that allows it to inherit the `h3` styles, because in that case it becomes the section head.

Reply:
yeah, I agree. If we had class-based styles this would be a much simpler change to make. It's also possible that the tier 1 agency header shouldn't be an h2 at all. I'm going to leave this one alone for now.